### ISSUE TITLE
Test for WCAG success criterion 1.4.4:  Resize text

### ISSUE THREAD

**Issue by hannolans** [quailjs/quail#209]
http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-scale.html
Except for captions and images of text, text can be resized without assistive technology up to 200 percent without loss of content or functionality. (Level AA) 
## Applicable

Always
## Failures
- [ ] F69 ( #222): Resizing visually rendered text up to 200 percent causes the text, image or controls to be clipped, truncated or obscured
- [ ] F80: Text-based form controls do not resize when visually rendered text is resized up to 200% 
## Techniques
- [ ] G142 (#221): Using a technology that has commonly-available user agents that support zoom
- [ ] C12: Using percent for font sizes
- [ ] C13: Using named font sizes
- [ ] C14: Using em units for font sizes
- [ ] SCR34: Calculating size and position in a way that scales with text size (Scripting)
- [ ] G146: Using liquid layout 
- [ ] G178: Providing controls on the Web page that allow users to incrementally change the size of all text on the page up to 200 percent
- [ ] G179: Ensuring that there is no loss of content or functionality when the text resizes and text containers do not change their width 
## Relationship
- G142 will always pass, so no further technique checks needed for minimal testing
- F69 is under discussion, implement as soon aswe have more clarity
- F80 is only applicable for forms
